examples/generate_image_dataset/user_config.yaml.example

```diff
@@ -19,6 +19,19 @@ SIMULATION_CONTROLLER:
   TASK: "NullTask"
   CUSTOM_UNREAL_CONSOLE_COMMANDS: []
 
+  NAVMESH:
```

Comment:
I don't think this navmesh stuff needs to be here because these are somewhat low-level default parameters that most users won't need to care about. My approach to including default parameters in `user_config.yaml.example` files so far is to include them only when I especially want to highlight the parameter. For example, the default for `FOV` is already `90.0` but I like that this config value is in `user_config.yaml.example` because it's helpful for the reader.

Reply:
Removed all Navmesh default params (as set in default_config.simulation_controller.yaml)